language_model/tensorflow2/bert/README.md

```diff
@@ -0,0 +1,130 @@
+# Location of the input files 
+
+This [GCS location](https://console.cloud.google.com/storage/browser/pkanwar-bert) contains the following.
+* TensorFlow1 checkpoint (bert_model.ckpt) containing the pre-trained weights (which is actually 3 files).
+* Vocab file (vocab.txt) to map WordPiece to word id.
+* Config file (bert_config.json) which specifies the hyperparameters of the model.
+* TensorFlow2 checkpoint should be convertable from the Tensorflow1 checkpoint using the checkpoint_converter:
+
+```shell
+python3 tf2_encoder_checkpoint_converter.py \
+  --bert_config_file=<path to bert_config.json> \
+  --checkpoint_to_convert=<path to tf1 model.ckpt-28252> \
+  --converted_checkpoint_path=<path to output tf2 model checkpoint>
+```
+
+Alternatively, the tf2 checkpint is also available [here](https://pantheon.corp.google.com/storage/browser/nnigania_perf_profiles/bert_mlperf_data). Note that the checkpoint converter removes optimizer slot variables, so the resulted tf2 checkpoint is only about 1/3 size as the tf1 checkpoint.
+
+# Download and preprocess datasets
+
+Download the [wikipedia dump](https://dumps.wikimedia.org/enwiki/20200101/enwiki-20200101-pages-articles-multistream.xml.bz2) and extract the pages
+The wikipedia dump can be downloaded from this link in this directory, and should contain the following file:
+enwiki-20200101-pages-articles-multistream.xml.bz2
+
+Run [WikiExtractor.py](https://github.com/attardi/wikiextractor) to extract the wiki pages from the XML
+The generated wiki pages file will be stored as <data dir>/LL/wiki_nn; for example <data dir>/AA/wiki_00. Each file is ~1MB, and each sub directory has 100 files from wiki_00 to wiki_99, except the last sub directory. For the 20200101 dump, the last file is FE/wiki_17.
+
+Clean up
+The clean up scripts (some references here) are in the scripts directory.
+The following command will run the clean up steps, and put the results in ./results
+./process_wiki.sh '<data dir>/*/wiki_??'
+
+After running the process_wiki.sh script, for the 20200101 wiki dump, there will be 500 files, named part-00xxx-of-00500 in the ./results directory.
+
+Exact steps (starting in the bert path)  
+
+```shell
+cd cleanup_scripts  
+mkdir -p wiki  
+cd wiki  
+wget https://dumps.wikimedia.org/enwiki/20200101/enwiki-20200101-pages-articles-multistream.xml.bz2    # Optionally use curl instead  
+bzip2 -d enwiki-20200101-pages-articles-multistream.xml.bz2  
+cd ..    # back to bert/cleanup_scripts  
+git clone https://github.com/attardi/wikiextractor.git  
```

Comment:
This current master version may be broken. See https://github.com/mlcommons/training/issues/429

Reply:
Thanks for pointing that out.
Let me double check what version I used when generating the inputs, and add the commit hash here.